### PR TITLE
Adding collection of Quicksight users and group

### PR DESCRIPTION
**Pull request by SohamMajumder**
*Issue #, if available:*
https://github.com/awslabs/cid-framework/issues/174

*Description of changes:*
Adding Quicksight Module to collect Quicksight related information from Management and Linked Accounts extending the data collection framework. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: data-collection/test/test_from_scratch.py [hunk 1]
```diff
@@ -175,10 +175,17 @@ def test_license_manager_grants(athena):
     data = athena_query(athena=athena, sql_query='SELECT * FROM "optimization_data"."license_manager_grants" LIMIT 10;')
     assert len(data) > 0, 'license_manager_grants is empty'
 
-def test_license_manager_licenses(athena):
```
Comment (iakov-aws) [inline data-collection/test/test_from_scratch.py:178]: please can you check this? i do not think this is intentional

Comment (SohamMajumder) [reply data-collection/test/test_from_scratch.py]: Fixed now. Thanks for the catch. Yes it got overwritten during earlier commit

File: data-collection/deploy/module-quicksight.yaml [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+          #Authors: Soham Majumder
+          """ Lambda function to collect Quicksight User, Groups, Membership across
+              Accounts and Namespaces
+          """
+          import os
+          import json
+          import logging
+          import datetime
+          from json import JSONEncoder
+          import boto3 #pylint: disable=C0413
```
Comment (iakov-aws) [inline data-collection/deploy/module-quicksight.yaml:114]:
```suggestion
          import boto3
```

File: data-collection/deploy/module-quicksight.yaml [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+              logger.info("Collecting data for account: %s", account_id)
+              quicksight_client = assume_role(account_id, "quicksight", "us-east-1") # must be us-east-1
+              namespaces = list_namespaces(account_id=account_id, quicksight=quicksight_client)
+
+              users = []
+              groups = []
+              group_memberships = []
+
+              for namespace in namespaces:
+                  capacity_region=namespace['CapacityRegion']
```
Comment (iakov-aws) [inline data-collection/deploy/module-quicksight.yaml:168]:
```suggestion
                  capacity_region = namespace['CapacityRegion']
```

File: data-collection/deploy/module-quicksight.yaml [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+                  # quicksight_client=quicksight_client
```
Comment (iakov-aws) [inline data-collection/deploy/module-quicksight.yaml:169]:
```suggestion

```

**Comment by iakov-aws** [comment]
continue in https://github.com/awslabs/cid-framework/pull/212

**Pull request closed without merging**